scripts/create-component/plop-templates-component/src/components/{{componentName}}/{{componentName}}.types.ts.hbs

```diff
@@ -1,40 +1,12 @@
 import * as React from 'react';
-import { ComponentProps, ShorthandProps } from '@fluentui/react-utils';
-import { ColorTokens, RecursivePartial, FontTokens } from '@fluentui/theme';
+import { ComponentProps } from '@fluentui/react-utils';
 
-/**
- * {@docCategory {{componentName}}\}
- */
```

Comment:
I think that these docstrings are needed for conformance tests, since that metadata is acted on.

However, I never understood the trailing `\` and TS will always warn about that